app/app.js

```diff
@@ -37,16 +41,23 @@ app.use(express.json())
 
 //custom errors
 
+app.use((err,req,res,next)=>{
+  if(err.msg){
+    res.status(err.status).send({msg: err.msg})
+    next()
```

Comment:
this next won't be doing anything i don't think.  You will have already sent your error response, so there is not need for it.